Update create-device-platform-restrictions.md (#3963)
MicrosoftDocs/memdocs: memdocs/intune/enrollment/create-device-platform-restrictions.md

## Create a device platform restriction   

1. Sign in to the [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431).  

> 2. Go to **Devices**, Uner policies click on **Enrollment device platform restrictions**.  

Reviewer: ```suggestion
2. Go to **Devices** and under **Policies**, select **Enrollment device platform restrictions**.  
```
